src/routes/wallet/p2sh/tx-proposal.routes.js

```diff
@@ -81,6 +81,12 @@ txProposalRouter.post(
       isString: true,
       optional: true,
     },
+    mark_inputs_as_used: {
+      in: ['body'],
+      errorMessage: 'Invalid mark_as_used argument',
+      isBoolean: true,
```

Comment:
Should we have `toBoolean: true` here as well?

Reply:
Done